src/Microsoft.Data.SqlClient/src/Interop/Windows/Kernel32/Interop.FreeLibrary.netcore.cs

```diff
@@ -2,6 +2,8 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+#if !NETFRAMEWORK
```

Comment:
```suggestion
#if NET
```

Reply:
Potato potato, I suppose... I thought I had some kind of rationale for it. Maybe that if we ever drop support for netfx, it'll be easier to find instances of NETFRAMEWORK vs NET (since that's used in a lot more?). Eh. I'll do a sweep and make the changes.